akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcSunnyWeatherSpec.scala

```diff
@@ -0,0 +1,164 @@
+class MultiDcSunnyWeatherMultiJvmNode2 extends MultiDcSunnyWeatherSpec
+class MultiDcSunnyWeatherMultiJvmNode3 extends MultiDcSunnyWeatherSpec
+class MultiDcSunnyWeatherMultiJvmNode4 extends MultiDcSunnyWeatherSpec
+class MultiDcSunnyWeatherMultiJvmNode5 extends MultiDcSunnyWeatherSpec
+//class MultiDcSunnyWeatherMultiJvmNode6 extends MultiDcSunnyWeatherSpec
+//class MultiDcSunnyWeatherMultiJvmNode7 extends MultiDcSunnyWeatherSpec
+
+abstract class MultiDcSunnyWeatherSpec extends MultiNodeSpec(MultiDcSunnyWeatherMultiJvmSpec)
+  with MultiNodeClusterSpec {
+
+  import MultiDcSunnyWeatherMultiJvmSpec._
+
+  import ClusterEvent._
+
+  "A normal cluster" must {
+    "be healthy" taggedAs LongRunningTest in {
+
+      val alphaObserver = TestProbe("alpha-observer")
+      val betaObserver = TestProbe("beta-observer")
+
+      // allow all nodes to join:
+      awaitClusterUp(roles: _*)
+
+      val crossDcHeartbeatSenderPath = "/system/cluster/core/daemon/crossDcHeartbeatSender"
+      val selectCrossDcHeartbeatSender = system.actorSelection(crossDcHeartbeatSenderPath)
+
+      val expectedAlphaHeartbeaterNodes = cluster.takeNOldestMembers(_.datacenter == "alpha", 2)
+      val expectedAlphaHeartbeaterRoles = membersAsRoles(expectedAlphaHeartbeaterNodes)
+
+      val expectedBetaHeartbeaterNodes = cluster.takeNOldestMembers(_.datacenter == "beta", 2)
+      val expectedBetaHeartbeaterRoles = membersAsRoles(expectedBetaHeartbeaterNodes)
+
+      val expectedNoActiveHeartbeatSenderRoles = roles.toSet -- (expectedAlphaHeartbeaterRoles union expectedBetaHeartbeaterRoles)
+
+      enterBarrier("found-expectations")
+
+      info(s"expectedAlphaHeartbeaterNodes = ${expectedAlphaHeartbeaterNodes.map(_.address.port.get)}")
+      info(s"expectedBetaHeartbeaterNodes = ${expectedBetaHeartbeaterNodes.map(_.address.port.get)}")
+      info(s"expectedNoActiveHeartbeatSenderRoles = ${expectedNoActiveHeartbeatSenderRoles.map(_.port.get)}")
+
+      runOn(expectedAlphaHeartbeaterRoles.toList: _*) {
+        selectCrossDcHeartbeatSender ! CrossDcHeartbeatSender.ReportStatusTo(Some(alphaObserver.ref))
+        val status = alphaObserver.expectMsgType[CrossDcHeartbeatSender.MonitoringActive](10.seconds)
+        info("Monitoring status: " + status)
+      }
+      runOn(expectedBetaHeartbeaterRoles.toList: _*) {
+        selectCrossDcHeartbeatSender ! CrossDcHeartbeatSender.ReportStatusTo(Some(betaObserver.ref))
+        val status = alphaObserver.expectMsgType[CrossDcHeartbeatSender.MonitoringActive](10.seconds)
+        info("Monitoring status: " + status)
+      }
+      runOn(expectedNoActiveHeartbeatSenderRoles.toList: _*) {
+        selectCrossDcHeartbeatSender ! CrossDcHeartbeatSender.ReportStatusTo(Some(betaObserver.ref))
+        val status = alphaObserver.expectMsgType[CrossDcHeartbeatSender.MonitoringDormant](10.seconds)
+        info("Monitoring status: " + status) // here we expect that they do NOT monitor anything
+      }
```

Comment:
so basically we're testing that the "2 oldest on each dc, are actually monitoring (active)" and "the other nodes, are dormant"